flare_portal/experiments/templates/experiments/participant_list.html

```diff
@@ -25,20 +25,31 @@ <h3 class="card-title">Participants</h3>
                                 <p class="card-subtitle m-0 ml-2">{{ participants|length }} participant{{ participants|length|pluralize }}</p>
                             </div>
                             <div class="card-options">
-                                <a href="{% url "experiments:participant_upload" project_pk=view.kwargs.project_pk experiment_pk=view.kwargs.experiment_pk %}" class="btn btn-sm btn-secondary">
+                                <a href="{% url "experiments:participant_upload" project_pk=view.kwargs.project_pk experiment_pk=view.kwargs.experiment_pk %}" class="btn btn-sm btn-secondary ml-4"">
                                     Upload
                                 </a>
                                 <a href="{% url "experiments:participant_create_batch" project_pk=view.kwargs.project_pk experiment_pk=view.kwargs.experiment_pk %}" class="btn btn-sm btn-secondary ml-4">
                                     Generate batch
                                 </a>
-                                <button class="btn btn-sm btn-primary ml-4" type="submit">Save changes</button>
+                                <button x-show="selectedRows.length == 0" class="btn btn-sm btn-primary ml-4" type="submit">Save changes</button>
+                                <button
+                                    x-show="selectedRows.length > 0"
+                                    x-text="selectedRows.length > 1 ? `Delete ${selectedRows.length} participants` : `Delete participant`"
+                                    class="btn btn-sm btn-danger ml-4"
+                                    type="button"
+                                    @click="
+                                        window.location = `{% url 'experiments:participant_delete_batch' project_pk=view.kwargs.project_pk experiment_pk=view.kwargs.experiment_pk participant_pks='PKS-PLACEHOLDER' %}`
+                                            .replace('PKS-PLACEHOLDER', selectedRows.join())
```

Comment:
I think this needs URL encoding as well for the commas? Or use a URL-safe separator, `-` or `_`.

```suggestion
                                            .replace('PKS-PLACEHOLDER', encodeURIComponent(selectedRows.join()))
```

Reply:
Thanks!